engine/chain/internal/bhs/get_merkleroots_from_bhs.go

```diff
@@ -0,0 +1,64 @@
+package bhs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"syscall"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/errors"
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+	"github.com/bitcoin-sv/spv-wallet/models"
+)
+
+// GetMerkleRootsFromBHS returns Merkle Roots from Block Header Service
+func (s *Service) GetMerkleRootsFromBHS(ctx context.Context, query url.Values) (*models.MerkleRootsBHSResponse, error) {
+	bhsURL, err := s.createBHSURL("/chain/merkleroot")
+	if err != nil {
+		return nil, err
+	}
+
+	req := s.httpClient.R().
+		SetContext(ctx).
+		SetHeader("Content-Type", "application/json").
+		EnableTrace()
```

Comment:
I'm not convinced if we need the `EnableTrace` here. I didn't apply it for other `resty`-requsts in the `chain` package. 
hmmm 🤔 

Maybe we could rather use [client.EnableTrace](https://pkg.go.dev/github.com/go-resty/resty/v2#Client.EnableTrace) for the whole resty client - and decide whether or not to use it on the level where the client is created ?

I'm wondering how this influence to performance. Perhaps we should enable trace only when `logger level is set to debug` 🤔

Reply:
Yeah, you're right it should rather be set for the client and not per request. I will remove it from here and I suggest we discuss it if we even want it set or in a particular scenario as you mentioned